### PR TITLE
Update Set-SPOTenant.md

### PR DESCRIPTION
**Pull request by maazarif11**
Added a new parameter AllowWebPropertyBagUpdateWhenNoScriptIsOn

### CODE REVIEW

**Comment by maazarif11** [comment]
@microsoft-github-policy-service agree company="Microsoft"

**Review by samkabue — CHANGES_REQUESTED** [review]
:clock1:

File: sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md [hunk 1]
```diff
@@ -167,6 +167,7 @@ Set-SPOTenant
  [-PrebuiltModelScope <SyntexFeatureScopeValue>]
  [-PrebuiltModelSelectedSitesList [String[]]]
  [-PrebuiltModelSelectedSitesListOperation <SelectedSitesListOperations>]
+ [-AllowWebPropertyBagUpdateWhenNoScriptIsOn <Boolean>]
```
Comment (samkabue) [inline sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md:170]: >AllowWebPropertyBagUpdateWhenNoScriptIsOn [](http://example.com/codeflow?start=3&length=41)

putting a wait pending the ongoing discussion for the backend PR

Comment (maazarif11) [reply sharepoint/sharepoint-ps/sharepoint-online/Set-SPOTenant.md]: @samkabue changed the name of the parameter based on the comments on the backend PR. Please let me know what you think.

**Review by samkabue — APPROVED** [review]
:shipit:

**Review by patrick-rodgers — APPROVED** [review]
Looks great, thanks!

**Comment by maazarif11** [comment]
@samkabue / @patrick-rodgers do you know someone that can help with completing this PR as I don't have write access to merge it.

**Comment by maazarif11** [comment]
Hi @wlibebe / @pamgreen-msft, can you please help with merging this PR? Thanks!

**Comment by pamgreen-msft** [comment]
Signed off, ready for Magic Skilling PubOps team to merge in the next window this afternoon.

**Pull request merged**